src/components/forms/AddEmployeeForm.js

```diff
@@ -17,8 +17,9 @@ function AddEmployeeForm({ visible, onCreate, onCancel }) {
         form
           .validateFields()
           .then(values => {
-            form.resetFields();
             onCreate(values);
+            form.resetFields();
+            console.log(values);
```

Comment:
For some reason I was under the impression that console.logs should not be pushed to production.